### PR TITLE
Updated text for mozilla donate help page

### LINKED ISSUE #1668: [DONATE] Update copy on donor care page
**As a donor care rep, I want to add copy to our donor help page so that supporters can find out to get the right kind of help from us**

Please edit donor [help page](https://donate.mozilla.org/en-CA/help/) with copy to be provided by @Stephanie Lowery

By end of May 2022

### PR DESCRIPTION
**Pull request by danielfmiranda**
Closes #1668

This PR updates the Mozilla donate sites help page copy from:

> If you need help with a donation to the Mozilla Foundation, please fill out this form and a donor care representative will get back to you as soon as possible.
> 
> Unfortunately, donor care representatives are unable to offer support or help with Firefox technical issues. For technical support, please visit the [Firefox support page](https://support.mozilla.org/questions/firefox).

To:

> If you need help with a donation to the Mozilla Foundation, please fill out this form and a donor care representative will get back to you as soon as possible.
> 
> Unfortunately, donor care representatives are unable to offer support or help with Firefox technical issues, or changes to or questions about Mozilla VPN subscriptions. Related questions submitted through this form will not be responded to.
> 
> For technical support, please visit the [Firefox support page](https://support.mozilla.org/questions/firefox). For help with a VPN subscription, please visit the [Mozilla VPN Support page](https://support.mozilla.org/products/firefox-private-network-vpn)
> 

Screenshots:
---
New Copy:

Screen Shot 2022-07-05 at 4 28 37 PM

### CODE REVIEW

**Review by TheoChevalier — CHANGES_REQUESTED** [review]
Thanks @danielfmiranda, just need to remove some extra spaces, otherwise looks good!

File: donate/templates/pages/core/contributor_support_page_master.html [hunk 1]
```diff
@@ -69,8 +69,13 @@ <h1 class="heading heading--primary">
...
-                                Unfortunately, donor care representatives are unable to offer support or help with Firefox technical issues. For technical support, please visit the <a href="{{ support_url }}">Firefox support page</a>.
+                            {% blocktrans trimmed %}
+                                Unfortunately, donor care representatives are unable to offer support or help with Firefox technical issues, or changes to or questions about Mozilla VPN subscriptions. Related questions submitted through this form will not be responded to.
+                            {% endblocktrans %}
+                        </p>
+                        <p class="rich-text">
+                            {% blocktrans with firefox_support_url="https://support.mozilla.org/questions/firefox" vpn_support_url="https://support.mozilla.org/products/firefox-private-network-vpn" trimmed %}
+                                For technical support, please visit the <a href="{{ firefox_support_url }}"> Firefox support page </a>. For help with a VPN subscription, please visit the <a href="{{ vpn_support_url }}"> Mozilla VPN Support page </a>.
```
[the remark this answers is not in the document]
Comment (danielfmiranda) [reply donate/templates/pages/core/contributor_support_page_master.html]: Thanks for the suggestion @TheoChevalier!

I have updated the repo to remove the spaces and re-requested review 👍

**Review by TheoChevalier — APPROVED** [review]
Thanks!

**Comment by anilkanji** [comment]
Thanks all - this looks good and our stakeholder is pleased!

**Pull request merged**